Championgg/dist/presence.js

```diff
@@ -11,22 +11,26 @@ presence.on("UpdateData", async () => {
         presenceData.details = "Initial page";
         presenceData.state = "Viewing champions";
         presenceData.startTimestamp = time;
+        presence.setActivity(presenceData);
     }
     else if (path === "/statistics/") {
         presenceData.details = "Statistics";
         presenceData.state = "Viewing win rates";
         presenceData.startTimestamp = time;
+        presence.setActivity(presenceData);
     }
     else if (path.startsWith("/matchup")) {
         presenceData.details = "Analysing matchup";
         presenceData.state = "Preparing to fight";
         presenceData.startTimestamp = time;
+        presence.setActivity(presenceData);
     }
     else if (path.startsWith("/champion")) {
         const name = path.replace("/champion/", "");
         presenceData.details = "Checking Runes";
         presenceData.state = name;
         presenceData.startTimestamp = time;
+        presence.setActivity(presenceData);
     }
     else if (presenceData.details == null) {
```

Comment:
else if (presenceData.details == null) {

This shouldn't be else if, it should be just a standalone if

So change it to that and remove the setactivitys you added and see if it works then

Reply:
Code has been corrected, thank you for the tips :D